pkgs/top-level/all-packages.nix

```diff
@@ -13870,6 +13870,7 @@ with pkgs;
   qemu_xen_4_10-light = lowPrio (qemu.override { hostCpuOnly = true; xenSupport = true; xen = xen_4_10-light; });
 
   qemu_test = lowPrio (qemu.override { hostCpuOnly = true; nixosTestRunner = true; });
+  qemu_test_gtk = lowPrio (qemu.override { hostCpuOnly = true; nixosTestRunner = true; gtkSupport = true; });
```

Comment:
Why do you change `qemu_test =` instead of `qemu`?

Reply:
`qemu_test` is the variant used by nixos tests and `nixos-build-vms`, so it is convenient to have a prebuilt version of this with gtk support.  Of course one could add `qemu_gtk` and `qemu_kvm_gtk` but I didn't want to create too many variants in `all_packages.nix`.

Reply:
Instead of creating more variants, we could also consider enabling gtk support by default. Probably just a question of closure size...

Reply:
That patch overrides some `9p` permission checks and should not be used in production for security reasons, so we do need a separate `qemu_test`. Anyway, why don't we keep it simple and enable `gtk3` support by default (except for `qemu-xen` which never needs graphics and should be kept small)? We can add variants later as needed.